coverage/.last_run.json

```diff
@@ -0,0 +1,5 @@
+{
+  "result": {
+    "covered_percent": 100.0
+  }
```

Comment:
collapse all following coverage files, THEY'RE LENGTHY & UNIMPORTANT (coverage was 100% tho, dont even worry about it)

Reply:
Wasnt even worried, then I was, now Im not! Boom!

Reply:
Gotta get that 100% COVERAGE